### DISCUSSION TITLE
User issues.

### DISCUSSION THREAD

**Discussion by IslandTrish** [OpenHoldem/openholdembot#45] [User Issues]
Starting the user issues section!

**Comment by morussul** [comment]
Hi, just connecting to 888 poker using scraper and the latest tablemap seems to be about six years old? is anyone willing to share a more recent 8 max table map?

**Reply by IslandTrish** [reply]
I don't have a newer version, maybe try asking on the MIM forum someone else might.

http://www.maxinmontreal.com/
